_posts/2020-10-13-bigsur-new.md

```diff
@@ -0,0 +1,330 @@
+
+# What has changed on the surface
+
+## A whole new iOS-like UI
+
+Love it or hate it, we've got a new UI more reminiscent of iOS 14 with hints of skeuomorphism(A somewhat subtle call back to previous mac UIs which have neat details in the icons)
+
+## macOS Snapshotting
+
+A feature initially baked into APFS back in 2017 with the release of macOS 10.13, High Sierra, now macOS's main System volume has become both read-only and snapshotted. What this means is:
+
+* 3rd parties have a much more difficult time modifying the system volume, allowing for greater security
+* OS updates can now be installed while you're using the OS, similar to how iOS handles updates
+* Time Machine can now more easily perform backups, without file inconsistencies with HFS Plus while you were using the machines
+
+However there are a few things to note with this new enforcement of snapshotting:
+
+* OS snapshots are not calculated as used space, instead being labeled as purgeable space
+* Disabling macOS snapshots for the root volume with break software updates, and can corrupt data if one is applied
+
+
+# What has changed under the hood
+
+Quite a few things actually! Both in good and bad ways unfortunately.
+
+* [New Kernel cache system: KernelCollections!](#new-kernel-cache-system-kernelcollections)
+* [New Kernel Requirements](#new-kernel-requirements)
+* [Broken Kexts in Big Sur](#broken-kexts-in-big-sur)
+* [MSI Navi installer Bug Resolved](#msi-navi-installer-bug-resolved)
+* [New AMD OS X Kernel Patches](#new-amd-os-x-patches)
+* [Other notable Hackintosh issues](#other-notable-hackintosh-issues)
+  * [Several SMBIOS have been dropped](#several-smbios-have-been-dropped)
+  * [Dropped hardware](#dropped-hardware)
+  * [Extra long install process](#extra-long-install-process)
+  * [X79 and X99 Boot issues](#x79-and-x99-boot-issues)
+  * [New RTC requirements](#new-rtc-requirements)
+  * [SATA Issues](#sata-issues)
+
+## New Kernel Cache system: KernelCollections!
+
+So for the past 15 years, macOS has been using the Prelinked Kernel as a form of Kernel and Kext caching. And with macOS Big Sur's new Read-only, snapshot based system volume, a new version of caching has be developed: KernelCollections!
+
+How this differs to previous OSes:
+
+* Kexts can no longer be hot-loaded, instead requiring a reboot to load with `kmutil`
+* The Secure Boot and standard kernel are now one(ie. no more dedicated Immutable Kernel)
+* Symbols are no longer required
+
+The last point is the most important part, as this is what we use for kext injection in OpenCore. Currently Apple has left symbols in place seemingly for debugging purposes however this is a bit worrying as Apple could outright remove symbols in later versions of macOS. But for macOS 11, Big Sur's cycle, we'll be good on that end however we'll be keeping an eye on macOS 12.
```

Comment:
```suggestion
The last point is the most important part, as this is what we use for kext injection in OpenCore. Currently Apple has left symbols in place seemingly for debugging purposes however this is a bit worrying as Apple could outright remove symbols in later versions of macOS. But for macOS 11, Big Sur's cycle, we'll be good on that end however we'll be keeping an eye on future releases of macOS.
```

Reply:
> Apple can drop support for symbols in macOS 11.x, not necessary 12

Very unlikely, as Apple rarely touches the their kernelcache system within an OS cycle and in all honesty wouldn't make sense especially if devs would be relying on them as well